float32 Curvature
float32 Heading
float32 Left_Dist
float32 Right_Dist
uint8 Left_Type
uint8 Right_Type
uint8 Left_Conf
uint8 Right_Conf
uint8 Left_LDW_avai
uint8 Right_LDW_avai
uint8 Constr_Area
float32 YawAngle
float32 PitchAngle

